vagrant/provisioning/roles/minio/tasks/main.yml

```diff
@@ -0,0 +1,66 @@
+
+- name: create minio group
+  become: yes
+  group:
+    name: minio
+    state: present
+
+- name: create minio user
+  become: yes
+  user:
+    name: minio
+    group: minio
+    shell: /sbin/nologin
+    home: /home/minio
+
+- name: create minio bin directory
+  become: yes
+  file:
+    path: "{{root_folder}}/app/minio/{{ item }}"
+    state: directory
+    owner: minio
+  with_items:
+    - "bin"
+    - "data"
+
+- name: check if minio file exists
```

Comment:
The download task in common module checks whether the file exists and downloads if necessary.  Use that task, instead of these two tasks (`stat` and `get_url`)

Reply:
Sure, we will merge this

Reply:
@Bharat-DRS , don't forget to switch to the shared download task.

Reply:
Changed the download task and now using the Common task to download the minio file

Reply:
@Bharat-DRS I see you are still using `get_url`, and not using the download task at all.  The download task would look like this;

```
- include_tasks: "{{ role_path }}/../common/tasks/download.yml"
  loop:
    - name: "MariaDB JDBC Driver"
      dest: "{{ root_folder }}/common/mariadb-java-client-{{ mariadb_jdbc_versio
n }}.jar"
      owner: "root"
      url: "https://downloads.mariadb.com/Connectors/java/connector-java-{{ mari
adb_jdbc_version }}/mariadb-java-client-{{ mariadb_jdbc_version }}.jar"
```

This is an example from the mariadb-client role.